src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -182,21 +186,42 @@ def build_graph_from_python_source(self, python_code: str) -> list[DependencyPro
 
     def _process_node(self, base_node: NodeBase):
         if isinstance(base_node, SysPathChange):
-            self._mutate_path_lookup(base_node)
-        if isinstance(base_node, NotebookRunCall):
-            strpath = base_node.get_notebook_path()
-            if strpath is None:
-                yield DependencyProblem('dependency-not-constant', "Can't check dependency not provided as a constant")
-            else:
-                yield from self.register_notebook(Path(strpath))
-        if isinstance(base_node, ImportSource):
-            prefix = ""
-            if isinstance(base_node.node, ImportFrom) and base_node.node.level is not None:
-                prefix = "." * base_node.node.level
-            name = base_node.name or ""
-            yield from self.register_import(prefix + name)
+            yield from self._mutate_path_lookup(base_node)
+        elif isinstance(base_node, NotebookRunCall):
+            yield from self._register_notebook(base_node)
+        elif isinstance(base_node, ImportSource):
+            yield from self._register_import(base_node)
+        else:
+            logger.error(f"Can't process {NodeBase.__name__} of type {type(base_node).__name__}")
```

Comment:
```suggestion
            logger.warning(f"Can't process {NodeBase.__name__} of type {type(base_node).__name__}")
```
Error logs fail the workflow

Reply:
done